src/main/resources/StoreSchema

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS category,property,item,catalog;
```

Comment:
In order to satisfy [requirements](https://github.com/ChangeRequest/jdbc.hw1/blob/master/README.md#3-store-relations) from [README](https://github.com/ChangeRequest/jdbc.hw1/blob/master/README.md):

> 1. One item should contain many properties. Many items may contain the same properties.
> 2. One item should be related to many categories. Many items may be related to many categories.
> 3. One item may be related to many catalogs. One catalog may contains many items.

 you **have to** create additional tables for relations.